Add bicep for IaC

wviriya opened the pull request:
## Purpose
<!-- Describe the intention of the changes being proposed. What problem does it solve or functionality does it add? -->
Add Bicep

## Does this introduce a breaking change?
<!-- Mark one with an "x". -->
```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

<!-- Please check the one that applies to this PR using "x". -->
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code
<!-- Add steps to run the tests suite and/or manually test -->
```
```

## What to Check
Verify that the following are valid
* ...

## Other Information
<!-- Add any other helpful information that may be needed here. -->

wviriya commented:
@DanielLarsenNZ just added Bicep files for IaC. I will let you do the merging.

DanielLarsenNZ commented:
Thanks Victor! Should `vulsCount.txt` be ignored?
Can you add a deployment script please?

wviriya commented:
> Thanks Victor! Should `vulsCount.txt` be ignored? Can you add a deployment script and instructions in `setup.md` please?

Updated setup.md and removed vulsCount.txt
